ledger-tool/src/main.rs

```diff
@@ -381,7 +383,18 @@ fn output_account(
     println!("  rent_epoch: {}", account.rent_epoch());
     println!("  data_len: {}", account.data().len());
     if print_account_data {
-        println!("  data: '{}'", bs58::encode(account.data()).into_string());
+        if encoding == UiAccountEncoding::Base58 {
```

Comment:
Looks like this is only handling base58 and jsonParsed?
I expected to see a complete match statement here:
```
match encoding {
    UiAccountEncoding::JsonParsed => {...}
    UiAccountEncoding::Base64 => {...}
    UiAccountEncoding::Base64Zstd => {...}
    _ => {...}
}
```

Reply:
It handles all the cases. it uses json serializers for all other cases except for base58. For base58,  to bypass the data size limit for encoding this condition is used.

Reply:
Oh, duh, I follow. Thanks